### PR TITLE
Implement --outputs_as_nchw option to transpose output to from nhwc to nchw

### PR DESCRIPTION
**Pull request by hwangdeyu**
fixes https://github.com/onnx/tensorflow-onnx/issues/1973 https://github.com/onnx/tensorflow-onnx/issues/1986
Add a option to make outputs format nhwc to nchw.
tf2onnx only supports `--inputs-as-nchw`  for specifying input as nchw so far. However, in some cases, such as models loaded by tensorRT or OpenCV DNN, it also needs to force output to nchw.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,7 +307,8 @@ model_proto, external_tensor_storage = tf2onnx.convert.from_keras(model,
         custom_rewriter: list of custom graph rewriters
         extra_opset: list of extra opset's, for example the opset's used by custom ops
         shape_override: dict with inputs that override the shapes given by tensorflow
-        inputs_as_nchw: transpose inputs in list from nchw to nhwc
+        inputs_as_nchw: transpose inputs in list from nhwc to nchw
+        outputs_as_nchw: transpose inputs in list from nhwc to nchw
```
Comment (fatcat-z) [inline README.md:311]: outputs?

File: README.md [hunk 2]
```diff
@@ -339,7 +340,8 @@ model_proto, external_tensor_storage = tf2onnx.convert.from_function(function,
...
```
Comment (fatcat-z) [inline README.md:344]: outputs?

File: README.md [hunk 3]
```diff
@@ -369,7 +371,8 @@ model_proto, external_tensor_storage = tf2onnx.convert.from_graph_def(graph_def,
...
```
Comment (fatcat-z) [inline README.md:375]: outputs?

File: README.md (hunk 1 above)
Comment (hwangdeyu) [reply README.md]: Yes, fixed.

**Review by fatcat-z — APPROVED** [review]
LGTM, thanks!

**Pull request merged**